packages/opentelemetry-core/src/utils/environment.ts

```diff
@@ -85,6 +87,8 @@ export const DEFAULT_ENVIRONMENT: Required<ENVIRONMENT> = {
   OTEL_EXPORTER_JAEGER_ENDPOINT: '',
   OTEL_EXPORTER_JAEGER_PASSWORD: '',
   OTEL_EXPORTER_JAEGER_USER: '',
+  OTEL_EXPORTER_PROMETHEUS_HOST: '',
+  OTEL_EXPORTER_PROMETHEUS_PORT: 9464,
```

Comment:
I wonder why this is part of core as it effects only the PrometheusExporter.
I would expect that all 3rd party configurations are moved to the corresponding 3rd party modules.

I assume the exporter should be compatible with other SDK implementations following the Otel API and exporter interface.